src/main/pages/user-management/user-management.adoc

```diff
@@ -8,67 +8,102 @@ folder: user-management
 ---
 
 
-[id="auth-and-user-management"]
-== Auth and User Management
+[id="authorization-and-user-management"]
+== Authorization and user management
 
-Eclipse Che relies on http://www.Keycloak.org[Keycloak] to create, import, manage, delete and authenticate users. Keycloak uses its own authentication mechanisms and user storage. Eclipse Che requires a Keycloak token when access to Che resources is requested.
+Eclipse Che uses http://www.Keycloak.org[Keycloak] to create, import, manage, delete, and authenticate users.  Keycloak uses built-in authentication mechanisms and user storage.  It can use third-party identity management systems to create and authenticate users. Eclipse Che requires a Keycloak token when you request access to Che resources.
 
-Keycloak can create and authenticate users by itself or rely on 3rd party identity management systems and providers.
+Local users and imported federation users must have an email address in their profile.
 
-Note that it is required for all types of users (local or imported from federation) to have the email address field set. Che cannot work with users that have no email in their profile.
+The default Keycloak credentials are `admin:admin`. You can use the `admin:admin` credentials when logging into Che for the first time. It has system privileges. 
 
-Default Keycloak credentials are `admin:admin`. You can find your Keycloak URL either in OpenShift web console > keycloak namespace (if you deployed to OpenShift) of go to `$CHE_HOST:5050/auth` if Che is running on Docker. Admin user is also a pre-defined Che user with privileges on the system scope. So, you may use `admin:admin` credentials when loggin in to Che for the first time.
+To find your Keycloak URL:
 
-[id="che-and-keycloak"]
-== Che and Keycloak
+If Che is deployed on OpenShift:
 
-When Che is deployed on OpenShift or installed on Docker, either deployment script or CLI makes sure that Keycloak is properly configured. When a `che-public` client is created, there are two important fields to be populated:
+* Go to OpenShift web console > keycloak namespace 
```

Comment:
If these are GUI elements, they should be in bold. If not, then rather not use `>` as it suggests menu navigation.

Reply:
Sentence period.